ibm_cloud.gemspec

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+lib = File.expand_path("lib", __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "ibm_cloud_sdk_core/version"
+
+Gem::Specification.new do |spec|
+  spec.name = "ibm_cloud_sdk_core"
+  spec.version = IBMCloudSdkCore::VERSION
+  spec.authors = ["Mamoon Raja"]
+
+  spec.summary = "Official client library to use the IBM Cloud servicess"
```

Comment:
This is not a library to use the core services. It's probably a core library but it doesn't have any specific service information. There is also a typo